MATLAB/sensor_reading.m

```diff
@@ -102,7 +108,7 @@
             Y(j) = voltages(i);
         end
     end
-    if j >= 4  % TODO : Perhaps play with this parameter
+    if j >= 3  % TODO : Perhaps play with this parameter
```

Comment:
What was the motivation for changing this value? I'm not saying I did an extensive dig into why I set it at 4 and not 3, but I was more comfortable having the system over-constrained. Especially when the sun vector is towards the docking end of the satellite (otherwise we'd always have 4+ sensors in view of the sun anyway).

Reply:
I was just messing around with it.

Reply:
I can try and dig up some old MATLAB scripts that I made to play around with it. From what I remember, the error was bounded much better with (about 5x better) with the threshold set to 4 and not 3. Although either way the standard div was `< 2` degrees or so.

Reply:
Below are some example pointing errors with 3 and 4

## 3
![pointing_error_j_3](https://user-images.githubusercontent.com/39104088/69765687-ee322980-1142-11ea-8f2a-b8a7624acb8b.jpg)

## 4
![pointing_error_j_4](https://user-images.githubusercontent.com/39104088/69765690-f12d1a00-1142-11ea-9323-fb15bf00d34a.jpg)

Reply:
It's the minimum number of photodiodes above threshold to calculate a sun vector on the ADCS box.